addons/weather/functions/fnc_getMapData.sqf

```diff
@@ -48,30 +48,17 @@ GVAR(currentHumidity) = 0;
 GVAR(currentOvercast) = 0;
 
 // Get all non inherited arrays to filter maps that inherit from Stratis/Altis/Tanoa
-private _nonInheritedArrays = configProperties [configFile >> "CfgWorlds" >> _worldName, "isArray _x", false];
+private _cfgPath = (configFile >> "CfgWorlds" >> _worldName);
+private _nonInheritedArrays = configProperties [_cfgPath, "isArray _x", false];
 // And check if any custom non-inherited weather is defined through config and use that if so
-if ((configFile >> "CfgWorlds" >> _worldName >> "ACE_TempDay") in _nonInheritedArrays) exitWith {
-    if (isArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_TempDay")) then {
-        GVAR(TempDay) = getArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_TempDay");
-    };
-    if (isArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_TempNight")) then {
-        GVAR(TempNight) = getArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_TempNight");
-    };
-    if (isArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_Humidity")) then {
-        GVAR(Humidity) = getArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_Humidity");
-    };
-    if (isArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_WindSpeedMin")) then {
-        GVAR(WindSpeedMin) = getArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_WindSpeedMin");
-    };
-    if (isArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_WindSpeedMean")) then {
-        GVAR(WindSpeedMean) = getArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_WindSpeedMean");
-    };
-    if (isArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_WindSpeedMax")) then {
-        GVAR(WindSpeedMax) = getArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_WindSpeedMax");
-    };
-    if (isArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_WindDirectionProbabilities")) then {
-        GVAR(WindDirectionProbabilities) = getArray (configFile >> "CfgWorlds" >> _worldName >> "ACE_WindDirectionProbabilities");
-    };
+if ((_cfgPath >> "ACE_TempDay") in _nonInheritedArrays) exitWith {
+    if (isArray (_cfgPath >> "ACE_TempDay"))                    then { GVAR(TempDay) = getArray (_cfgPath >> "ACE_TempDay");};
```

Comment:
Keep the original indentation and formatting:
```suggestion
    if (isArray (_cfgPath >> "ACE_TempDay")) then {
        GVAR(TempDay) = getArray (_cfgPath >> "ACE_TempDay");
    };
```

I really dislike this type of formatting, because it's squished. What bothers me particularly with this formatting is that you have two semicolons on the same line, as well as a space as a the first character inside the `then {}` block, but no space on the other side. It's also unsymmetrical which makes me sad.